models/parrot_neuron.cpp

```diff
@@ -51,18 +51,16 @@ parrot_neuron::update( Time const& origin, const long_t from, const long_t to )
   assert( to >= 0 && ( delay ) from < Scheduler::get_min_delay() );
   assert( from < to );
 
-  SpikeEvent se;
-
   for ( long_t lag = from; lag < to; ++lag )
   {
     const ulong_t current_spikes_n = static_cast< ulong_t >( B_.n_spikes_.get_value( lag ) );
-
     if ( current_spikes_n > 0 )
     {
-      for ( ulong_t i_spike = 0; i_spike < current_spikes_n; i_spike++ )
-      {
-        network()->send( *this, se, lag );
-      }
+      // create a new SpikeEvent and set its multiplicity
+      SpikeEvent se;
+      se.set_multiplicity( current_spikes_n );
+
+      network()->send( *this, se, lag );
       set_spiketime( Time::step( origin.get_steps() + lag + 1 ) );
```

Comment:
https://github.com/nest/nest-simulator/issues/77#issuecomment-148698719 seems to suggest that this is indeed the case.

this would result in 
```c++
for (ulong_t i=0;  i<current_spikes_n; i++ )
    set_spiketime( Time::step( origin.get_steps() + lag + 1 ) );
```
with which we're almost back to the old form -- although it's a little cleaner with the handling of multiplicity.

It should be noted that the [original model](https://github.com/flinz/nest-simulator/blob/c229212d987023246b2fd7b1f072680b33e79e40/models/parrot_neuron.cpp) did not set the spike_times multiply.